### ISSUE TITLE
The "ratio" option in panel 

### ISSUE THREAD

**Issue by ti987** [automat/controlkit.js#10]
I am not sure how to change label:comp width ratio from the description. 
   	Number 	The ratio of label (default:30%) and component (default:70%) width
Does it take a number, a string, 2 numbers, or something else?
I tried a few different values but none of them worked.

**Comment by mootari** [comment]
I don't think this is actually implemented (maybe it was at some point). Searching for "ratio" doesn't reveal any use of the .ratio property. The only thing that can be found is the SCSS preset that defines a ratio of 40/60.

**Comment by polyclick** [comment]
+1, found out this property doesn't do anything :p
